### PR TITLE
Report the used port for the application.

### PR DESCRIPTION
**Pull request by obino**

### CODE REVIEW

**Comment by obino** [comment]
OCD run: https://ocd.appscale.com:8080/job/Daily%20Build/346/

File: AppController/djinn.rb [hunk 1]
```diff
@@ -1016,9 +1016,24 @@ def status(secret)
...
         next if !is_loaded
         next if app_name == "none"
+        stats_str << "    Information for application #{app_name}:\n"
+        stats_str << "        number of AppServers: "
         if !@app_info_map[app_name]['appengine'].nil?
-          stats_str << "    The number of AppServers for app #{app_name} is: " +
-            "#{@app_info_map[app_name]['appengine'].length}\n"
+          stats_str << "#{@app_info_map[app_name]['appengine'].length}\n"
+        else
+          stats_str << "unkonwn\n"
```
Comment (cdonati) [inline AppController/djinn.rb]: typo: "unknown"

File: AppController/djinn.rb [hunk 2]
```diff
@@ -1016,9 +1016,24 @@ def status(secret)
...
+        stats_str << "    Information for application: #{app_name}\n"
+        stats_str << "        number of AppServers: "
         if !@app_info_map[app_name]['appengine'].nil?
-          stats_str << "    The number of AppServers for app #{app_name} is: " +
-            "#{@app_info_map[app_name]['appengine'].length}\n"
+          stats_str << "#{@app_info_map[app_name]['appengine'].length}\n"
+        else
+          stats_str << "unknown\n"
+        end
+        stats_str << "        http port           : "
```
Comment (menivaitsi) [inline AppController/djinn.rb]: http -> HTTP

File: AppController/djinn.rb [hunk 3]
```diff
@@ -1016,9 +1016,24 @@ def status(secret)
...
+        if !@app_info_map[app_name]['nginx'].nil?
+            stats_str << "#{@app_info_map[app_name]['nginx']}\n"
+        else
+          stats_str << "unknown\n"
+        end
+        stats_str << "        https port          : "
```
Comment (menivaitsi) [inline AppController/djinn.rb]: https -> HTTPS

File: AppController/djinn.rb [hunk 4]
```diff
@@ -1016,9 +1016,24 @@ def status(secret)
       stats['apps'].each { |app_name, is_loaded|
         next if !is_loaded
         next if app_name == "none"
+        stats_str << "    Information for application: #{app_name}\n"
+        stats_str << "        number of AppServers: "
```
Comment (menivaitsi) [inline AppController/djinn.rb]: number -> Number

**Comment by obino** [comment]
Latest OCD run: https://ocd.appscale.com:8080/job/Daily%20Build/348/

**Comment by obino** [comment]
latest OCD run: https://ocd.appscale.com:8080/job/Daily%20Build/349/

**Comment by menivaitsi** [comment]
:+1: 

Let's do a demo before we merge.

**Pull request merged**